asab/library/service.py

```
@@ -382,6 +386,122 @@ async def _list(self, path, providers):
+			if before == after:
+				continue
+			added = sorted(list(after - before))
+			removed = sorted(list(before - after))
+			try:
+				self.App.PubSub.publish("Library.change!", {
+					"kind": "favorite",
+					"path": path,
+					"added": added,
+					"removed": removed,
+				})
+			except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)